database/migrations/1734288366467_create_guide_articles_table.ts

```diff
@@ -0,0 +1,22 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "guide_articles";
+
+  async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments("id").notNullable();
+      table.text("title").notNullable();
+      table.text("short_desc").notNullable();
+      table.text("description").notNullable();
+      table.text("image_path").notNullable();
+
+      table.timestamp("created_at");
+      table.timestamp("updated_at");
```

Comment:
timestamp columns should be made not nullable
```suggestion
      table.timestamp("created_at").notNullable();
      table.timestamp("updated_at").notNullable();
```